src/main/java/edu/usf/cutr/gtfs_realtime/bullrunner/GtfsRealtimeTripUpdatesProducerDemoMain.java

```diff
@@ -15,77 +15,47 @@
  */
 package edu.usf.cutr.gtfs_realtime.bullrunner;
 
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.FileReader;
-import java.io.IOException;
-import java.net.URL;
-import java.util.HashSet;
-import java.util.Set;
-
+import com.google.inject.Guice;
+import com.google.inject.Inject;
+import com.google.inject.Injector;
+import com.google.inject.Module;
 import org.apache.commons.cli.CommandLine;
 import org.apache.commons.cli.GnuParser;
 import org.apache.commons.cli.Options;
 import org.apache.commons.cli.Parser;
-import org.json.JSONException;
 import org.onebusaway.cli.CommandLineInterfaceLibrary;
 import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeFileWriter;
+import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeGuiceBindingTypes.TripUpdates;
+import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeGuiceBindingTypes.VehiclePositions;
 import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeServlet;
 import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeSource;
-
-import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeGuiceBindingTypes;
-import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeGuiceBindingTypes.VehiclePositions;
-import org.onebusaway.gtfs_realtime.exporter.GtfsRealtimeGuiceBindingTypes.TripUpdates;
-import org.onebusaway.guice.jsr250.JSR250Module;
 import org.onebusaway.guice.jsr250.LifecycleService;
 
-import com.google.inject.Guice;
-import com.google.inject.Inject;
-import com.google.inject.Injector;
-import com.google.inject.Module;
+import java.io.File;
+import java.net.URL;
+import java.util.HashSet;
+import java.util.Set;
 
 public class GtfsRealtimeTripUpdatesProducerDemoMain {
-
     private static final String ARG_TRIP_UPDATES_PATH = "tripUpdatesPath";
-
     private static final String ARG_TRIP_UPDATES_URL = "tripUpdatesUrl";
-
     private static final String ARG_VEHICLE_POSITIONS_PATH = "vehiclePositionsPath";
-
     private static final String ARG_VEHICLE_POSITIONS_URL = "vehiclePositionsUrl";
     private GtfsRealtimeProviderImpl _provider;
-    //	private BullRunnerConfigExtract _providerConfig;
     private LifecycleService _lifecycleService;
     private GtfsRealtimeSource _tripUpdates;
-    //	@Inject
-//	public void setProvider(BullRunnerConfigExtract providerConfig) {
-//		_providerConfig = providerConfig;
-//	}
     private GtfsRealtimeSource _vehiclePositions;
 
     public static void main(String[] args) throws Exception {
         GtfsRealtimeTripUpdatesProducerDemoMain m = new GtfsRealtimeTripUpdatesProducerDemoMain();
         m.run(args);
     }
 
-/*
```

Comment:
Looking at this more, unless we know exactly what these @Inject methods are doing I'd suggest leaving them - I think they are an important part of the lifecycle.